StartInspection.py

```diff
@@ -10,11 +10,8 @@
 
 
 def is_bv_valid_for_plugin(bv: bn.BinaryView) -> bool:
-    if bv.arch.name == "x86_64" or bv.arch.name == "x86":
+    if bv.arch and (bv.arch.name == "x86_64" or bv.arch.name == "x86"):
         return True
-    else:
-        print(f'ClassyPP: Executable CPU Arch is: {bv.arch.name}. This plugin supports only x86 32/64 bit executables.')
-        return False
```

Comment:
Did you mean to remove the `return False`?

Reply:
Hmm, I don't think so, I think I was planning on just moving it to the top level but looks like it got eaten. It _might_ default to false on the core side so it might have just worked accidentally, but agree leaving it is best.